src/api/views.py

```diff
@@ -37,6 +37,72 @@ def get(self, request):
                 status=501,
             )
 
+# View 3
+class VictimGenders(generics.ListCreateAPIView):
+    def get(self, request, *args, **kwargs):
+        c_id = request.GET.get("community_id")
+        case_set = Cases.objects.filter(community_id=1).select_related('victim_id')
+        genders_to_counts = {
+            0: 0,
+            1: 0,
+            2: 0,
+            3: 0
+        }
+        total_count = 0
+        for case in case_set:
+            victim = case.victim_id
+            try:
+                genders_to_counts[victim.gender] += 1
+            except:
+                genders_to_counts[victim.gender] = 1
+            total_count += 1
+
+        counts = {
+            'Female': genders_to_counts[1],
+            'Male': genders_to_counts[2],
+            'Other': genders_to_counts[0] + genders_to_counts[3],
+            'Total Count': total_count
+        }
+        return JsonResponse(counts)
+
+# View 4
+class VictimEthnicities(generics.ListCreateAPIView):
```

Comment:
This one looks great! The reason we don't do what we did above is because ethnicities is stored as an array field and there isn't great support for get_gender_display() for array fields.